services/orchest-webserver/requirements.in

```
@@ -1,25 +1,28 @@
-nbconvert~=6.0.0
-# ipython is required for consistent HTML rendering, see:
+-e file:../../lib/python/orchest-internals
 # https://github.com/jupyter/nbconvert/issues/1443
+# ipython is required for consistent HTML rendering, see:
```

Alternatively,

```suggestion
ipython  # required for consistent HTML rendering, see https://github.com/jupyter/nbconvert/issues/1443
```

to accommodate alphabetical sorting. Don't think we should worry about line length here

---

Great suggestion, added in: https://github.com/orchest/orchest/pull/1134/commits/2b11dd6a6135deb8e694fbac7df2015ed51f04c9